pkgs/dash_analytics/README.md

```diff
@@ -0,0 +1,160 @@
+This package is intended to be used on Dash (Flutter, Dart, etc.) related tooling only.
+It provides APIs to send events to Google Analytics using the Measurement Protocol.
+
+## Usage
```

Comment:
The 'how to use' docs here are great!

In order to help messaging around who should use this tool, you may consider:
- shortening this file to just a "what's this" section and a pointer to docs for using the library
- move the `Usage` section to a separate file

Something like:

```
## What's this?

This is an internal package used by Dart and Flutter tooling; it is not intended to be general purpose or consumed by the community (it makes assumptions about file names, workflows, ...).

It provides APIs to send events to Google Analytics using the Measurement Protocol.

## Using this library

See ...
```